torch/_torch_docs.py

```diff
@@ -4191,6 +4242,15 @@ def merge_dicts(*dicts):
     {input}
 """.format(**common_args))
 
+add_docstr(torch.is_conj, r"""
+is_conj(input) -> (bool)
+
+Returns True if the :attr:`input` is a conjugated tensor, i.e. its conjugate bit is set to `True`.
+
```

Comment:
This is fine as-is; an addition might be to see also `conj()`, check out the cool work @Lezcano did adding "see also" boxes to linalg: https://pytorch.org/docs/master/generated/torch.linalg.svd.html#torch.linalg.svd